bitriseapi/bitriseapi.go

```diff
@@ -39,6 +39,13 @@ type TriggerAPIResponseModel struct {
 	TriggeredWorkflow string `json:"triggered_workflow"`
 }
 
+// SkipAPIResponseModel ...
+type SkipAPIResponseModel struct {
+	CommitHash    string `json:"commit_hash"`
```

Comment:
I'm not sure whether these params are important at all - the webhook's "request" info should include these, but might be a good idea.

What I'd definitely add is:

```
Message           string `json:"message"`
```

Which would include the description of why it was skipped